# HG changeset patch
# User VSpiridonov
# Parent  26508bce401a9321f4c39d777321139f34097e7b
refs #DEVBO-656+3  !!! отладка!!! Продажа Энергет. ГАЗ_бл.с возр.огранич.
изменено ref\imp-goods-1C-RN.p ( IS18Plus=1 - уст.атр., 0,отсутсвие,пустой тег - удал.атр.)

diff --git a/ref/imp-goods-1C-RN.p b/ref/imp-goods-1C-RN.p
--- a/ref/imp-goods-1C-RN.p
+++ b/ref/imp-goods-1C-RN.p
@@ -503,8 +503,10 @@ end.
   then do:
     RUN gds-attr-write (v-nbc, {&attr-IS18Plus}, p-GdsObj:IS18Plus).
   end.  
-  else do:
-    RUN gds-attr-delete (v-nbc, {&attr-IS18Plus}, output v-attr-del).
+
+  if p-GdsObj:IS18Plus = 0 or p-GdsObj:IS18Plus = ? 
+  then do:
+     RUN gds-attr-delete (v-nbc, {&attr-IS18Plus}, output v-attr-del).
   end.  
 
  
